### ISSUE TITLE
`no-magic-numbers` should ignore parseFloat radix param

### ISSUE THREAD

**Issue by saveman71** [eslint/eslint#10445]
For additional context, please see #4167 first :)

**What rule do you want to change?**
`no-magic-numbers`.

**Does this change cause the rule to produce more or fewer warnings?**
Fewer.

**How will the change be implemented? (New option, new default behavior, etc.)?**
New default behavior is to ignore `parseFloat` radix as well. A new option would be nice to disable the behavior for `parseInt` and `parseFloat`. I suggest a boolean named `ignoreRadixes` that defaults to `true`.

**Please provide some example code that this change will affect:**

```js
const newValue = parseFloat(value, 10);
```

**What does the rule currently do for this code?**
Produces the warning "No magic number: 10".

**What will the rule do after it's changed?**
Produce no warning.

---

I've already worked on this, PR referenced below.

**Comment by Aladdin-ADD** [comment]
Thanks for the issue! I'm not sure `parseFloat` has a radix param. is there some change in es6+?

**Comment by saveman71** [comment]
Whoops, looks like I've made a fool of myself. The sample code was on an existing codebase, so I assumed it existed without checking.

Sorry for the time taken.

Closing.
